docs/release-process.md

```diff
@@ -80,22 +80,22 @@ to date too.  You will need to have a PyPI account and access to the sambacc
 repo.
 
 Log into PyPI web UI. (Re)Generate a pypi login token for sambacc.
-Ensure `twine` is installed:
+Ensure `twine` is installed (consider installing it in a fresh virtual env):
 ```
 python3 -m pip install --upgrade twine
 ```
 
 Create a directory to store the python build artifacts:
 ```
-rm -rf _build/pypi
-mkdir -p _build/pypi
-cp sambacc-0.3.tar.gz sambacc-0.3-py3-none-any.whl _build/pypi
+mypypi=$PWD/_builds/v03/pypi
```

Comment:
```suggestion
mypypi=$PWD/_build/v03/pypi
```

Typo or intentional?

Reply:
This is intentional. The name was inconsistent with the section above, now it's consistent.

Reply:
I see what you mean, but it seems to be  a bit more subtle to me:

Above, I see both `$PWD/_builds/v03/pypi` and  `.
Bu I see that the  `nypypi`variable assignment is now consistent.

Thanks for clarifying.